projects/plugins/jetpack/modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -2415,7 +2414,7 @@ protected function parseChunk() {
 		if (empty($this->buffer)) return false;
 		$s = $this->seek();
 
-		$block = null;
+		$block = null; // Initialize for wpcom's rector.
```

Comment:
Personally I'd still move this one down to just after line 2517. Or does rector find some other code path I'm not seeing?